src/components/OrderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import {ThemeProvider} from '../ThemeProvider';
+import {Card} from '../Card';
+import {TextContainer} from '../TextContainer';
+import {Thumbnail} from '../Thumbnail';
+import {Stack} from '../Stack';
+import {Button} from '../Button';
+import {TextStyle} from '../TextStyle';
+
+export interface OrderDetailsProps {}
+
+export class OrderDetails extends React.PureComponent<OrderDetailsProps> {
+  render() {
+    return (
+      <ThemeProvider
+        theme={{
+          colorScheme: 'dark',
+          colors: {
+            surface: '#020c1d',
```

Comment:
This makes sense because we're using a specific color theme. There's also a way to import colors from polaris if there were other specific colors from the palette we wanted.